Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/TextInput.swift

```diff
@@ -106,6 +106,7 @@ private extension TextInput {
             }
             .focused($isFocused)
             .frame(maxWidth: .infinity, alignment: .leading)
+#if !os(visionOS)
             .toolbar {
                 ToolbarItemGroup(placement: .keyboard) {
                     if UIDevice.current.userInterfaceIdiom == .phone, isFocused, (element.fieldType?.isNumeric ?? false) {
```

Comment:
I am talking to David and Mark about what to do with this toolbar item on visionOS. It doesn't make sense to put it anywhere other than the keyboard which you can't because the keyboard placement isn't available on visionOS. I have another idea that we are discussing.

Reply:
Solution in this commit: https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/875/commits/7057555360e68692b2b587fc2523284ac54d61c6.

Discussed with Mark and David.